indico/htdocs/js/indico/widgets/protection_widget.js

```diff
@@ -29,16 +34,16 @@
         }, options);
 
         var inputs = $('input[name=' + options.fieldName + '][id^=' + options.fieldId + ']');
+        var $enableAclLink = $('.enable-acl-link');
```

Comment:
I think in Python it's much more common to keep acronyms all uppercase. At least we do it pretty much everywhere in the backend